docs/additional_information/credits.rst

```diff
@@ -4,38 +4,83 @@ Contact and Credits
 ===================
 
 If you have any questions or comments, please do not hesitate to contact us via filing
-an issue on Github or directly via email.
+an issue on Github, writing an `email`_ to our `zulipchat group
+<http://gettsim.zulipchat.com/>`_ or via personal emails.
 
-Development Lead
-^^^^^^^^^^^^^^^^
+.. _email: respy.9b46528f81292a712fa4855ff362f40f.show-sender@streams.zulipchat.com
+
+
+Project Manager
+---------------
 
 `Philipp Eisenhauer <https://github.com/peisenha>`_
 
-Contributors
-^^^^^^^^^^^^
 
-| `Janos Gabler <https://github.com/janosg>`_
-| `Boryana Ilieva <https://github.com/boryana-ilieva>`_
-| `Tobias Raabe <https://github.com/tobiasraabe>`_
+Software Design
+---------------
+
+- `Janos Gabler <https://github.com/janosg>`_
+- `Tobias Raabe <https://github.com/tobiasraabe>`_
+
+
+Developer
+---------
+
+- `Annica Gehlen <https://github.com/amageh>`_
+- `Moritz Mendel <https://github.com/mo2561057>`_
+
 
```

Comment:
Why no contributors as we discussed, or commiters? Rafael, Sofia ..

Reply:
I missed it. Done.